app/serializers/rest/account_serializer.rb

```diff
@@ -30,15 +30,7 @@ def moved?
     end
   end
 
-  class RoleSerializer < ActiveModel::Serializer
-    attributes :id, :name, :color
-
-    def id
-      object.id.to_s
-    end
-  end
-
-  has_many :roles, serializer: RoleSerializer, if: :local?
+  has_many :roles, serializer: REST::RoleSerializer, if: :local?
```

Comment:
A User can only have one Role set via the admin UI as far as I can tell. As we didn't expose `highlighted` here, I suspect this is why we didn't see User Roles get used in downstream client apps, meaning they couldn't be used for classification / grouping of users, e.g., "Corporate Account" or "OSS Project" (two "roles" Hachyderm has)